pkg/cmd/cluster/connect/connect.go

```diff
@@ -48,9 +53,6 @@ func NewConnectCommand(f *factory.Factory) *cobra.Command {
 		Example: opts.localizer.MustLocalize("cluster.connect.cmd.example"),
 		Args:    cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, _ []string) error {
-			if opts.ignoreContext == true && !opts.IO.CanPrompt() {
```

Comment:
Note - this requires documentation change and better documentation in the cluster root cmd